services/editor/src/components/UserBar.js

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from "react";
+import { mapPropsStream } from "recompose";
+import { Observable } from "rxjs";
+import fetch from "../utils/fetch";
+import md5 from "md5";
+import styled from "react-emotion";
+import { Link } from 'react-router-dom';
+
+const Container = styled("div")`
+    display: flex;
+    background-color: #00506D;
+    width: 166px;
+    justify-content: flex-end;
+    border-left: 1px solid #eee;
+    flex-direction: row;
+    height: 100%;
+    color: white;
+    align-items: center;
+    padding-right:10px;
+  `;
+
+export default mapPropsStream(props$=>
```

Comment:
can you please split to
``` js
const withCurrentUser = mapPropsStream...
const UserBar = (props) => ...
export default withCurrentUser(UserBar)
```

to improve readability and to add a display name to the component for easier debugging